### ISSUE TITLE
sadded

### ISSUE THREAD

**Issue by dabail10** [CICE-Consortium/Icepack#391]
Does anyone know why the sadded line in flood_ice (icepack_therm_mushy.F90) is commented out?

!      sadded = wadded * ice_ref_salinity * p001

**Comment by eclare108213** [comment]
@dabail10 you commented this (sadded) line out in PR #142 
https://github.com/CICE-Consortium/Icepack/commit/13558b206713051d25db23c39309b5896ca03288

**Comment by dabail10** [comment]
I guess the more relevant question is why is sadded not used anywhere?

**Comment by eclare108213** [comment]
Good point.  wadded is not used either, except for sadded.  eadded is used for an 'advective heat flux' associated with flooding.  I'm not sure why mass of water and salt are not treated the same way -- they might be captured elsewhere but it's not clear.  We are planning to work through all of these issues as part of the merge process.

**Comment by dabail10** [comment]
Did we resolve this?
